### ISSUE TITLE
Train IP Adapter 

### ISSUE THREAD

**Issue by AMohamedAakhil** [huggingface/diffusers#7194]
There isn't an example of how to train or finetune your own IP Adapter model currently. 
It would be helpful to have an implementation of the how to train/finetune the IP Adapter face id, sdxl and plus versions 

I'd like to work on this issue

**Comment by rootonchair** [comment]
It seems like the code to train IP-Adapter is provided in the original repo: https://github.com/tencent-ailab/IP-Adapter

**Comment by AMohamedAakhil** [comment]
Yes, I just wanted to add this within the diffusers documentation and provide an example with it too. Can I work on that?

**Comment by sapkun** [comment]
> Yes, I just wanted to implement this within diffusers, and update the docs. I was initially confused on how to exactly go about it, would be helpful to have an example in the docs. Can I work on this?

Hello! Have you completed integrating the IP adapter training script into the project?

**Comment by AMohamedAakhil** [comment]
still working on it!
https://github.com/huggingface/diffusers/pull/7196
